adapters/orbidder/orbidder.go

```diff
@@ -80,8 +88,25 @@ func preprocess(imp *openrtb2.Imp) error {
 	return nil
 }
 
+func preprocessBidFloorCurrency(imp *openrtb2.Imp, reqInfo *adapters.ExtraRequestInfo) error {
+	// we expect every currency related data to be EUR
+	if imp.BidFloorCur == "" {
+		imp.BidFloorCur = "EUR"
```

Comment:
Given that `imp.BidFloorCur` ends up being set to `"EUR"` in line 104 anyways, we could remove this `if` statement altogether.